subgraphs/hop-protocol/protocols/hop-protocol/config/deployments/hop-protocol-arbitrum/configurations.ts

```diff
@@ -374,13 +385,13 @@ export class HopProtocolArbitrumConfigurations implements Configurations {
     ];
   }
   getPoolAddressFromRewardTokenAddress(rewardToken: string): string {
-    if (rewardToken == ArbitrumRewardToken.DAI) return ArbitrumAmm.USDC;
-    else if (rewardToken == ArbitrumRewardToken.ETH) return ArbitrumAmm.USDT;
-    else if (rewardToken == ArbitrumRewardToken.USDC) return ArbitrumAmm.ETH;
-    else if (rewardToken == ArbitrumRewardToken.USDT) return ArbitrumAmm.ETH;
+    if (rewardToken == ArbitrumRewardToken.DAI) return ArbitrumAmm.DAI;
```

Comment:
Do you know if this was a mistake or was it supposed to be like that at one point?